arcusApp/app/src/main/java/arcus/app/subsystems/alarm/promonitoring/presenters/AlarmCardPresenter.java

```diff
@@ -1,318 +0,0 @@
-/*
- *  Copyright 2019 Arcus Project.
- *
- *  Licensed under the Apache License, Version 2.0 (the "License");
- *  you may not use this file except in compliance with the License.
- *  You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- *  Unless required by applicable law or agreed to in writing, software
- *  distributed under the License is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
- */
-package arcus.app.subsystems.alarm.promonitoring.presenters;
-
-import android.content.Context;
-import android.os.Handler;
-import android.os.Looper;
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
-
-import arcus.cornea.provider.DeviceModelProvider;
-import arcus.cornea.provider.HubModelProvider;
-import arcus.cornea.subsystem.SubsystemController;
-import arcus.cornea.subsystem.alarm.AlarmProviderController;
-import arcus.cornea.subsystem.alarm.AlarmSubsystemController;
-import arcus.cornea.subsystem.alarm.model.AlarmModel;
-import arcus.cornea.utils.Listeners;
-import arcus.cornea.utils.ModelSource;
-import com.iris.client.capability.AlarmSubsystem;
-import com.iris.client.capability.DeviceConnection;
-import com.iris.client.capability.Hub;
-import com.iris.client.capability.SecuritySubsystem;
-import com.iris.client.capability.SwannBatteryCamera;
-import com.iris.client.event.ListenerRegistration;
-import com.iris.client.model.DeviceModel;
-import com.iris.client.model.HubModel;
-import com.iris.client.model.ModelDeletedEvent;
-import com.iris.client.model.SubsystemModel;
-import arcus.app.R;
-import arcus.app.common.cards.SimpleDividerCard;
-import arcus.app.common.controller.AbstractCardController;
-import arcus.app.common.utils.StringUtils;
-import arcus.app.dashboard.settings.services.ServiceCard;
-import arcus.app.subsystems.alarm.promonitoring.cards.ProMonitoringDashboardCard;
-import arcus.app.subsystems.alarm.promonitoring.util.AlarmUtils;
-import arcus.app.subsystems.learnmore.cards.LearnMoreCard;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.beans.PropertyChangeEvent;
-import java.beans.PropertyChangeListener;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-
-public class AlarmCardPresenter extends AbstractCardController<SimpleDividerCard> implements AlarmSubsystemController.Callback,  AlarmProviderController.Callback {
```

Comment:
I think this file needs to stay - even though it's part of promonitoring, it appears to be the entire alarm view in the app, which explains why you can't control the alarm from the app

Reply:
Okay, I will make a flat alarm submodule without the promonitoring branding.

Reply:
On second thought, since there is still the promonitoring submodule with remnant promonitoring code that isnt interfering with the compile I will just restore this file.